MAINTAINERS.md

```diff
@@ -0,0 +1,37 @@
+# Podman Maintainers
+
+[GOVERNANCE.md](https://github.com/containers/podman/blob/main/GOVERNANCE.md)
+describes the Podman project's governance and the Project Roles used below.
+
+## Maintainers
+
+| Maintainer           | GitHub ID                                                | Project Roles     | Affiliation                               |
```

Comment:
table seems broken, it is not rendered correctly on github
https://github.com/containers/podman/blob/027818e4e4a610f6587d70fceadc86135aec8750/MAINTAINERS.md